charts/orchestrator/templates/postgres.yaml

```diff
@@ -0,0 +1,99 @@
+{{- if eq .Values.postgres.externalPostgresDBHost "" }}
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: postgres-db
+  namespace: {{ .Values.orchestrator.namespace }}
+  labels: 
+    app: postgres-db
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgres-db
+  template:
+    metadata:
+      labels:
+        app: postgres-db
+    spec:
+      containers:
+        - name: postgres-db
+          image: 'quay.io/ecosystem-appeng/rhdh-postgres'
+          imagePullPolicy: 'Always'
```

Comment:
Image pull policy should be consistent for all the image

Reply:
Corrected